BioscoopSysteemAPI/BioscoopSysteemAPI/Migrations/CinemaDbContextModelSnapshot.cs

```diff
@@ -30,9 +30,15 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     SqlServerPropertyBuilderExtensions.UseIdentityColumn(b.Property<int>("MovieId"));
 
+                    b.Property<bool>("Add3DMovie")
+                        .HasColumnType("bit");
+
                     b.Property<byte>("AllowedAge")
                         .HasColumnType("tinyint");
 
+                    b.Property<DateTime>("Date")
+                        .HasColumnType("datetime2");
```

Comment:
Wat is de reden dat er datetime2 staat? Bestaat er niet een normale datetime?

Reply:
Dat komt van de migratie geen idee :)